Packs/JWT/Integrations/JWT/JWT.yml

```
@@ -0,0 +1,243 @@
+category: Authentication
+commonfields:
+  id: JWT
+  version: -1
+configuration:
+- additionalinfo: The authentication server URL
```

```suggestion
- additionalinfo: The authentication server URL for example: https://example.com/auth/v2/token
```

maybe its better to add more information about the url

---

It's any JWT authentication service URL, not specific to any product, that why I kept it generic